dll/win32/rasdlg/lang/ro-RO.rc

```diff
@@ -282,15 +282,15 @@ BEGIN
     LTEXT "&Domeniul:", 1110, 10, 100, 92, 8
     EDITTEXT 1102, 104, 95, 154, 14, ES_UPPERCASE | ES_AUTOHSCROLL
     CONTROL "", -1, "STATIC", SS_LEFTNOWORDWRAP | SS_SUNKEN, 11, 116, 247, 1
-    AUTOCHECKBOX "Păstrea&ză acest nume și parolă pentru acești utilizatori:", 1101, 16, 124, 239, 13, BS_LEFT | BS_TOP | BS_MULTILINE | WS_GROUP
+    AUTOCHECKBOX "Sal&vare a acestor nume și parolă pentru acești utilizatori:", 1101, 16, 124, 239, 13, BS_LEFT | BS_TOP | BS_MULTILINE | WS_GROUP
     AUTORADIOBUTTON "Pentru mine, e&xclusiv", 1622, 29, 139, 225, 11, BS_LEFT | BS_TOP | BS_MULTILINE | WS_GROUP
     AUTORADIOBUTTON "Pentru &oricine din acest calculator", 1623, 29, 152, 225, 11, BS_LEFT | BS_TOP | BS_MULTILINE
     CONTROL "", -1, "STATIC", SS_LEFTNOWORDWRAP | SS_SUNKEN | WS_GROUP, 10, 167, 247, 1
     LTEXT "Apelea&ză:", 1412, 10, 177, 90, 8
     COMBOBOX 1416, 102, 173, 154, 112, CBS_DROPDOWN | CBS_AUTOHSCROLL | WS_VSCROLL | NOT WS_TABSTOP
     CONTROL "", -1, "STATIC", SS_LEFTNOWORDWRAP | SS_SUNKEN | WS_GROUP, 10, 192, 247, 1
-    DEFPUSHBUTTON "&Apelează", 1590, 10, 199, 60, 14, WS_GROUP
-    PUSHBUTTON "A&nulează", 1591, 72, 199, 60, 14
+    DEFPUSHBUTTON "&Apelare", 1590, 10, 199, 60, 14, WS_GROUP
+    PUSHBUTTON "&Revocare", 1591, 72, 199, 60, 14
```

Comment:
This change introduces an accelerator collision with the line below
PUSHBUTTON "P&roprietăți", 1107, 134, 199, 60, 14
Please check your romanian Windows to find out which of both lines need to be changed.